subworkflows/nf-core/abundance_differential_filter/main.nf

```
@@ -131,19 +130,41 @@ workflow ABUNDANCE_DIFFERENTIAL_FILTER {
 
     ch_versions = ch_versions.mix(PROPR_PROPD.out.versions.first())
 
+    // ----------------------------------------------------
+    // Run DREAM
+    // ----------------------------------------------------
+
+    // DREAM only runs with formula
+    ch_contrasts_with_formula = inputs.contrasts_for_diff
+        .filter{ it[0].method_differential == 'dream' }
+        .branch {
+            valid: it[0].formula != null
+            invalid: it[0].formula == null
+        }
+        .valid
```

Can we fail or at least warn on "invalid" inputs?